common/src/main/kotlin/entity/Permission.kt

```diff
@@ -9,13 +9,12 @@ import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
 import kotlinx.serialization.descriptors.SerialDescriptor
 import kotlinx.serialization.encoding.Decoder
 import kotlinx.serialization.encoding.Encoder
-import kotlin.contracts.ExperimentalContracts
 import kotlin.contracts.InvocationKind
 import kotlin.contracts.contract
 
 
 @Serializable(with = Permissions.Companion::class)
-class Permissions constructor(val code: DiscordBitSet) {
+data class Permissions constructor(val code: DiscordBitSet) {
```

Comment:
reason:
- data class to make `assertEquals` in ``GuildTest.`PartialGuild serialization`()`` pass
- data class seems appropriate
- `equals`/`hashCode`: `code` is the only property that differentiates `Permissions` instances (`values` is computed from `code`)